Tests/PlayMode/RotateDollhouseViewTest.cs

```diff
@@ -0,0 +1,232 @@
+        {
+            Assert.AreEqual(initialRotationX, dollhouseViewWrapper.localEulerAngles.x);
+            Assert.AreEqual(initialRotationY, dollhouseViewWrapper.localEulerAngles.y);
+
+            input.Press(keyboard.downArrowKey);
+            yield return new WaitForSeconds(timeInSecsForHoldingButton);
+            input.Release(keyboard.downArrowKey);
+            yield return null;
+
+            float expectedRotation = NormalizeAngle(initialRotationX - (keyboardRotationSpeedDegreesPerSecond * timeInSecsForHoldingButton));
+            Assert.AreEqual(expectedRotation, dollhouseViewWrapper.localEulerAngles.x, errorToleranceInDegrees);
+        }
+
+        [UnityTest]
+        public IEnumerator LeftArrowKeyShouldRotateDHVLeft()
+        {
+            Assert.AreEqual(initialRotationX, dollhouseViewWrapper.localEulerAngles.x);
+            Assert.AreEqual(initialRotationY, dollhouseViewWrapper.localEulerAngles.y);
+
+            input.Press(keyboard.leftArrowKey);
+            yield return new WaitForSeconds(timeInSecsForHoldingButton);
+            input.Release(keyboard.leftArrowKey);
+            yield return null;
+
+            float expectedRotation = NormalizeAngle(initialRotationY + (keyboardRotationSpeedDegreesPerSecond * timeInSecsForHoldingButton));
+            Assert.AreEqual(expectedRotation, dollhouseViewWrapper.localEulerAngles.y, errorToleranceInDegrees);
+        }
+
+        [UnityTest]
+        public IEnumerator RightArrowKeyShouldRotateDHVRight()
+        {
+            Assert.AreEqual(initialRotationX, dollhouseViewWrapper.localEulerAngles.x);
+            Assert.AreEqual(initialRotationY, dollhouseViewWrapper.localEulerAngles.y);
+
+            input.Press(keyboard.rightArrowKey);
+            yield return new WaitForSeconds(timeInSecsForHoldingButton);
+            input.Release(keyboard.rightArrowKey);
+            yield return null;
+
+            float expectedRotation = NormalizeAngle(initialRotationY - (keyboardRotationSpeedDegreesPerSecond * timeInSecsForHoldingButton));
+            Assert.AreEqual(expectedRotation, dollhouseViewWrapper.localEulerAngles.y, errorToleranceInDegrees);
+        }
+
+        [UnityTest]
+        public IEnumerator ShouldClampVerticalRotationToMaxVerticalRotationAngleWithKeyboard()
+        {
+            Assert.AreEqual(initialRotationX, dollhouseViewWrapper.localEulerAngles.x);
+            Assert.AreEqual(initialRotationY, dollhouseViewWrapper.localEulerAngles.y);
+
+            float angleToClamp = maxVerticalAngle + 5;
+            float timeToHoldHeyBeyondClamp = Mathf.Abs((angleToClamp - initialRotationX)) / keyboardRotationSpeedDegreesPerSecond;
+
+            input.Press(keyboard.upArrowKey);
+            yield return new WaitForSeconds(timeToHoldHeyBeyondClamp);
+            input.Release(keyboard.upArrowKey);
+
+            Assert.AreEqual(89.9f, dollhouseViewWrapper.localEulerAngles.x, errorToleranceInDegrees);
```

Comment:
there are several wild numbers on the loose here
```suggestion
            Assert.AreEqual(maxVerticalAngle, dollhouseViewWrapper.localEulerAngles.x, errorToleranceInDegrees);
```